app/src/main/java/com/nextcloud/talk/conversationlist/ConversationsListActivity.kt

```diff
@@ -345,6 +345,16 @@ class ConversationsListActivity :
         showSearchOrToolbar()
     }
 
+    override fun onPause() {
+        super.onPause()
+        val firstVisible = layoutManager?.findFirstVisibleItemPosition() ?: 0
+        val firstItem = adapter?.getItem(firstVisible)
+        val firstTop = (firstItem as ConversationItem).mHolder?.itemView?.top
+        val firstOffset = firstTop?.minus(CONVERSATION_ITEM_HEIGHT) ?: 0
+
+        appPreferences.setConversationListPositionAndOffset(firstVisible, firstOffset)
```

Comment:
@rapterjet2004 i moved this code to onPause, otherwise it would just work when coming back from a chat and not when the app was just in background.
Guess that's okay for you or was there a reason against onPause?